e2e/test/iothub/twin/TwinE2ETests.cs

```diff
@@ -713,24 +712,11 @@ private async Task Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGetDa
 
             var twinPatch = new Twin();
 
-            // Here is an example to create a TwinCollection with json object if the users use date-formatted
-            // string and don't want to drop the trailing zeros in the microseconds portion while parsing the
-            // json properties. The Newtonsoft.Json serializer settings added in SDK apply to serial-/deserialization
-            // during the client operations. Therefore, to ensure the datetime has the trailing zeros and this
-            // test can verify the returned peoperties not dropping them, manually set up "DateParseHandling.None"
-            // while creating a new TwinCollection object as the twin property.
-            JObject jobjectProperty = JsonConvert.DeserializeObject<JObject>(
```

Comment:
Previously, passing a date-formatted string with trailing zeros in a "TwinCollection" would drop those zeros. Turns out it was because `JObject.Parse(twinJson)` in "TwinCollection" had such a behavior and there's no workaround by adding serializer settings into Parse() call. So now, I am using `JsonConvert.DeserializeObject<JObject>()` there instead, and we don't have this problem any more.